docs/styles/demo/styles.css

```diff
@@ -1,13 +1,15 @@
 @layer
   reset,
   fonts,
+  tokens,
   typography,
   base,
   layout,
   third-party;
 
 @import url('../reset.css') layer(reset);
 @import url('../fonts.css') layer(fonts);
+@import url('/assets/packages/@rhds/tokens/css/global.css') layer(tokens);
 @import url('../typography.css') layer(typography);
 @import url('../third-party/prism-rhds.css') layer(third-party);
```

Comment:
@bennypowers I can't remember why we loaded prism into the demo styles.  This maybe something we can remove, especially with changes to codeblock, can you verify?